docs/source/guide/rem-5-theory.md

```diff
@@ -12,3 +12,27 @@ kernelspec:
 ---
 
 # What is the theory behind REM?
+
+Readout error mitigation (REM), is one of the most general and earliest studied error mitigation techniques, which can refer to a variety of specific approaches.
+
+A simple version of readout error mitigation is postselection of bitstrings. For
+example, if one knows that the measured bitstrings should preserve some
+symmetry, bitstrings that do not preserve it can be discarded. Such capability 
+is indeed available in {mod}`mitiq.rem.post_select`.
+
+With regards to the more elaborate technique of confusion matrix inversion,
+also [supported](rem-1-intro) in Mitiq, the most relevant references are Refs. {cite}`Maciejewski_2020,Bravyi_2021`.
+In Ref. {cite}`Maciejewski_2020`, a REM technique for NISQ devices has been
+introduced in conjuction with the reconstruction of a Positive-Operator Valued
+Measure (POVM) describing the measurement. The technique was there tested on a
+Rigetti quantum processor and a 5-qubit IBM quantum processor.
+
+In Ref. {cite}`Bravyi_2021` two REM schemes are introduced, based on tensor
+product noise and correlated Markovian noise models. The tecniques have been
+tested on the 20-qubit superconducting circuit processor ibmq-johannesburg.
+
+
+Other related work on REM is performed in Ref. {cite}`Garion_2021` (in
+conjuction with a custom randomized benchmarking technique) and in
+Ref. {cite}`Geller_2021` (to correct SPAM errors, and tested on an IBM Q
+processor with 4 and 8 superconducting qubits).
```

Comment:
Optional suggestion:

```suggestion
With regards to the more elaborate technique of confusion matrix inversion,
also [supported](rem-1-intro) in Mitiq, some relevant references are Refs. {cite}`Maciejewski_2020,Bravyi_2021,Garion_2021,Geller_2021`.
```

What about if we keep it as simple as this? At some point, not in this PR, it would be nice to have a proper theory section that explains how the technique works. But this is a good start. 

Some reasons for removing details:

- I am not sure the reconstruction of a Positive-Operator Valued Measure (POVM) describing the measurement is is a specific feature of Maciejewski_2020, it's a requirement of all confusion inversion methods.
- Experimental results  are probably outdated by more recent advanced experiments and don't fit too well in the theory section.